rustls-mio/examples/tlsserver.rs

```diff
@@ -568,8 +568,11 @@ fn make_config(args: &Args) -> Arc<rustls::ServerConfig> {
         NoClientAuth::new()
     };
 
-    let mut config = rustls::ServerConfig::new(client_auth);
-    config.key_log = Arc::new(rustls::KeyLogFile::new());
+    let suites = if !args.flag_suite.is_empty() {
+        lookup_suites(&args.flag_suite)
+    } else {
+        rustls::ALL_CIPHERSUITES.to_vec()
+    };
```

Comment:
Style nit: for short if-else conditionals like this, I find myself increasingly writing `match` expressions instead; these are a little bit more compact vertically and also usually nicely align the outcome values next to each other.